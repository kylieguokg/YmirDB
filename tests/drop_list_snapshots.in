drop 1
list snapshots
set a 28
set b 10
list entries
snapshot
list snapshots
del a
del b
set c 10
set d 10
list entries
snapshot
list snapshots
set e 19 2
set f 29 1
list entries
snapshot
list snapshots
set g e f
list entries
snapshot
list snapshots
drop 1
list snapshots
checkout 1
drop 3
list snapshots
checkout 3
drop 4
list snapshots
checkout 4
snapshot
checkout 5
list entries
bye
