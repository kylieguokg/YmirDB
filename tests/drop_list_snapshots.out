> no such snapshot

> no snapshots

> ok

> ok

> b [10]
a [28]

> saved as snapshot 1

> 1

> ok

> ok

> ok

> ok

> d [10]
c [10]

> saved as snapshot 2

> 2
1

> ok

> ok

> f [29 1]
e [19 2]
d [10]
c [10]

> saved as snapshot 3

> 3
2
1

> ok

> g [e f]
f [29 1]
e [19 2]
d [10]
c [10]

> saved as snapshot 4

> 4
3
2
1

> ok

> 4
3
2

> no such snapshot

> ok

> 4
2

> no such snapshot

> ok

> 2

> no such snapshot

> saved as snapshot 5

> ok

> g [e f]
f [29 1]
e [19 2]
d [10]
c [10]

> bye
